frontend/package.json

```diff
@@ -19,60 +19,58 @@
         "test-coverage": "jest --verbose --reporters=default --reporters=jest-junit --coverage"
     },
     "dependencies": {
-        "axios"               : "^0.19.2",
-        "bootstrap"           : "^4.4.1",
-        "bootstrap-icons"     : "1.0.0-alpha4",
-        "bootstrap-vue"       : "^2.13.0",
-        "@microsoft/signalr"  : "^5.0.0-preview.5.20279.2",
-        "vue"                 : "^2.6.11",
-        "@vue/composition-api": "^0.6.4"
+        "axios"               : "^0.21.1",
+        "bootstrap"           : "^4.6.0",
+        "bootstrap-icons"     : "^1.3.0",
+        "bootstrap-vue"       : "^2.21.2",
+        "@microsoft/signalr"  : "^5.0.3",
+        "vue"                 : "^2.6.12",
+        "@vue/composition-api": "^1.0.0-rc.1"
     },
     "devDependencies": {
-        "@babel/core"                       : "^7.10.2",
-        "@types/expect-puppeteer"           : "^4.4.3",
-        "@types/jest"                       : "^26.0.0",
-        "@types/jest-environment-puppeteer" : "^4.3.1",
-        "@types/jest-image-snapshot"        : "^3.1.0",
-        "@types/puppeteer"                  : "^3.0.0",
-        "@vue/test-utils"                   : "^1.0.3",
+        "@babel/core"                       : "^7.12.16",
+        "@types/expect-puppeteer"           : "^4.4.5",
+        "@types/jest"                       : "^26.0.20",
+        "@types/jest-environment-puppeteer" : "^4.4.1",
+        "@types/jest-image-snapshot"        : "^4.1.3",
+        "@types/puppeteer"                  : "^5.4.3",
```

Comment:
Looks like it's needed, as [CI fails](https://dev.azure.com/gh-gfoidl/AzureDevOpsTest/_build/results?buildId=1632&view=logs&j=1bf9a053-13aa-5fb9-9346-08fdf06e149a&t=6599ef50-6b8e-5459-4da7-071a78999b7e) otherwise. I don't know what type of TS support is anounced in puppeteer -- but TBH didn't check very well.